Dockerfile-ubi

```diff
@@ -1,9 +1,9 @@
 # Mondoo Multi-Architecture Container Dockerfile
 # 
 # To build with BuildX:   docker buildx build --build-arg VERSION=5.21.0 --platform 
-#             linux/386,linux/amd64,linux/arm/v7,linux/arm64 -t mondoolabs/mondoo:5.21.0 . --push
+#             linux/amd64,linux/arm64 -t mondoolabs/mondoo:5.21.0 . --push
 
-FROM redhat/ubi8
+FROM redhat/ubi8-minimal AS root
```

Comment:
Turns out ubi-minimal does not contain `yum`:
```
#0 0.267 /bin/sh: yum: command not found
```

From a redhat blog:
> The main difference between the ubi and ubi-minimal images is that the first provides the full yum toolset. Yum adds some dependencies such as Python packages. In contrast, the second provides microdnf as a replacement. The microdnf tool works from the same repositories as Yum, but only provides the ability to install, update, and delete packages:

https://developers.redhat.com/blog/2019/05/31/working-with-red-hat-enterprise-linux-universal-base-images-ubi#exploring_the_ubi_container_image

Reply:
fixed this